src/Logic/StripableText.cs

```diff
@@ -60,20 +63,24 @@ public StripableText(string text, string stripStartCharacters, string stripEndCh
                         text = text.Remove(0, endIndex);
                     }
                 }
+                while (text.Length < beginLength);
             }
 
             Post = string.Empty;
             if (text.Length > 0 && !Utilities.AllLettersAndNumbers.Contains(text[text.Length - 1]))
             {
-                for (int i = 0; i < 5; i++)
+                int beginLength;
+                do
                 {
+                    beginLength = text.Length;
+
                     while (text.Length > 0 && stripEndCharacters.Contains(text[text.Length - 1]))
                     {
                         Post = text[text.Length - 1] + Post;
                         text = text.Substring(0, text.Length - 1);
                     }
 
-                    if (text.EndsWith('>'))
+                    if (text.Length >= 4 && text[text.Length - 1] == '>')

```

Comment:
As usual I didn't get conclusive results ([from this test](https://gist.github.com/xylographe/561085cdea02c90ed0b5)).